homeassistant/components/local_ip/sensor.py

```diff
@@ -24,11 +24,21 @@ def name(self):
         """Return the name of the sensor."""
         return self._name
 
+    @property
+    def unique_id(self):
+        """Return the unique id of the sensor."""
+        return self._name
```

Comment:
There's precedence for defining a "fake" unique id for single instance entities:
https://github.com/home-assistant/core/blob/7ae802bb0db1169cb8a0264585c95cf70f7693ea/homeassistant/components/cloud/binary_sensor.py#L34-L37

Just make it a static string.

Reply:
Do you think the domain is fine ?

Reply:
We may want to add the sensor "type" if there will be more than one sensor in the future.